arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,552 @@
+          },
+          "204": {
+            "description": "No Content",
+            "schema": { "$ref": "#/definitions/RpProvisionResult" }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "description": "Updates the current state of the specified Analysis Services server as specified by the request body",
+        "tags": [ "RpProvision" ],
+        "operationId": "RpProvision_UpdateDetailsAsync",
+        "parameters": [
+          {
+            "name": "analysisServerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "name": "updateRequest",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/RpServerUpdateRequest" },
+            "description": "Request object for updating the server"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Name of the Azure Resource group"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/RpProvisionResult" }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers": {
+      "get": {
+        "description": "Gets all the AS servers for the given resource group",
+        "tags": [ "RpProvision" ],
+        "operationId": "RpProvision_GetAllServersForResourceGroupAsync",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Name of the Azure Resource group"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/ProvisionResultList" }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/servers": {
+      "get": {
+        "description": "Gets all the AS servers for the given subscription",
+        "tags": [ "RpProvision" ],
+        "operationId": "RpProvision_GetAllServersForSubscriptionAsync",
```

Comment:
Could you rename this operation id (and description) to "List" instead of "Get"? That will help indicate to customers that a set of results are returned, not just one entity